### DISCUSSION TITLE
[Ubuntu 22.04] Status, Outlook, What to do?

### DISCUSSION THREAD

**Discussion by WonkoMcSane** [azerothcore/azerothcore-wotlk#13642] [General]
Title pretty well summarizes it...

I've been seriously considering switching to AzerothCore from TrinityCore but I'm running Ubuntu22.04 on my server.

1) If openssl3 support is close then I can sit back and chill a bit longer.

2) If it's still looking to be a bit off- would someone mind linking a known-good 'how to compile openssl3 for ubuntu22.04'?  I'm not a Linux guru and I prefer to approach things in a controlled manner when making changes to my server. Sake of predictable results and all...

**Comment by Winfidonarleyan** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/13354

**Reply by WonkoMcSane** [reply]
Yeah... I really, really should have made mention of that, if for no other reason than to show that I /had/ done some degree of looking into it on my own...

I think the unstated part of my curiosity stems from having seen that, but not knowing what the development process is like around here... Looks like 'all systems go', but it's also been edging closer to 'a couple weeks' since the last activity.

I think I was soft-asking if there was any point to checking out that 'how to test AC PRs' link, or if you have what you need on that front and all that's left is background admin actions, I guess...? It's early, and before work mush brain.

**Comment by WonkoMcSane** [comment]
Wanted to comment that I went ahead and cloned the AC repo, applied the PR, compiled and auth/world seem to have run without error.

Made it as far as FirstRun database creation and population, but stopped there because the wife came home and wanted to play and I'm not sure what the current state of TC>AC migration is.

I expect to revisit to completion in the next couple days. When I do I'll add a confirmation to the PR, if it's still open.
